docs/video/index.md

```diff
@@ -5,27 +5,25 @@ icon: material/video-box
 
 # :material-video-box: Introduction
 
-This section of the wiki will show you how to optimally configure software to help you produce high-quality and optionally motion blurred videos with your own optimised personal workflow.
+This section of the wiki is for video-related content, specifically using OBS to record high-quality, high-framerate videos, and how motion blur can be applied to those videos.
 
 ## :material-book: Definitions
 
-### :material-run-fast: Frame blending
-The most common, it is without (unlike other methods like RSMB) any smearing of HUDs or general artifacting. Commonly this is done from a very high FPS (e.g. 540) down to a common FPS, i.e. 60 or 30.
+### :material-run-fast: Motion Blur
+When most people say motion blur, what they really mean is blending consecutive frames in a high-framerate video to a lower framerate, which replicates the effect of motion blur seen in real life and photography. This is called frame blending, and is what we'll be using in this wiki.
 
-In general, the higher the input FPS, the smoother the final output would be, due to more blur frames making the blur look more seamless. Blur frames are how many frames are blended together, so `blur frames = input fps / output fps`.
+In general, the higher the input FPS, the smoother the final output would be, since more blur frames (the frames combined to make the new frame) make the blur look more seamless. The number of blur frames can be calculated by dividing the input fps by the output fps, then multiplying by the blur amount: `blur frames = input fps / output fps * blur amount`.
 
-Video editors such as [Vegas Pro](/placeholder/) and [Adobe Premiere Pro](/placeholder/) have these features built-in. However, we recommend using seperate programs such as [blur](/placeholder/) first to pre-render the videos, therefore you don't have any lag in your video editor as well as having much more customisation.
+Video editors such as [Vegas Pro](/placeholder/) and [Adobe Premiere Pro](/placeholder/) have these features built-in. However, we recommend using seperate programs such as blur or smoothie first to pre-render the videos, since they're faster and more configurable with how they handle frame blending. Using them will also decrease preview lag in editors.
```

Comment:
> smoothie

should be capitalised

maybe also note that you can use interpolation, which will be described down below?

Reply:
👍 will add that